Modules/CalcSections-3_0.lua

```diff
@@ -475,6 +475,10 @@ return {
 		{ breakdown = "AreaOfEffectMod" }, 
 		{ modName = "AreaOfEffect", cfg = "skill" },
 	}, },
+	{ label = "Aura Effect", haveOutput = "AuraEffectMod", { format = "x {2:output:AuraEffectMod}",
```

Comment:
Change the label to "Aura Effect Mod", as that's consistent with the other mods that show up in this box.

Also, move it in the list. I'd put it below the duration mods, above the projectile mods. In this current spot, it splits up the AoE Mod and the Radius display, which just looks weird.

Reply:
Sure, done